admin_guide/cluster_autoscaler.adoc

```diff
@@ -0,0 +1,527 @@
+[[admin-guide-cluster-autoscaler]]
+= Managing the Cluster Auto Scaler
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[managing-autoscaler-overview]]
+== Overview
+
+This topic describes setting up the auto-scaler on your
+{product-title} cluster. The auto-scaler's remit is to ensure your
+pods have somewhere to run, and to also ensure there are no unneeded
```

Comment:
Are there more or specific reasons why someone with an AWS cluster would want to use ASGs?

Reply:
They provide elasticity for your application to automatically accommodate a growth in demand, and similarly to shrink when that demand ebbs. The ASG provides both an upper and lower bound; you may not want to consume 1000 nodes but may well be happy with 100. Equally, you can set the desired capacity above the minimum, accommodating the cost of having "a few" around, knowing there's some spare headroom to accommodate an increase in load/demand without having to wait for new nodes to be provisioned and join the cluster, typically this is 5+ minutes.